data/json/npcs/BG_traits.json

```diff
@@ -688,5 +692,58 @@
     "purifiable": false,
     "types": [ "BACKGROUND_SURVIVAL_STORY" ],
     "flags": [ "BG_SURVIVAL_STORY" ]
+  },
+  {
+    "type": "mutation",
+    "//": "This is a unique background story trait that should not be given out randomly.",
```

Comment:
Are any of these unique since I think they are all being distributed randomly above in the trait_group?

Reply:
Nope, just copied it over, good call